docs/src/content/docs/api/tools.mdx

```diff
@@ -0,0 +1,628 @@
+        <td>string</td>
+        <td>The name of the tool. Maximum 40 characters.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">definition<br /><span class="text-sm text-red-500">required</span></td>
+        <td>object</td>
+        <td>A [Tool](./#tool-definition-schema) definition.</td>
+      </tr>
+    </table>
+  </TabItem>
+  <TabItem label="Response">
+    <table>
+      <tr>
+        <th class="w-1/12"></th>
+        <th class="w-1/12"></th>
+        <th class="w-10/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">toolId</td>
+        <td>string</td>
+        <td>Unique identifier for the created tool.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">name</td>
+        <td>string</td>
+        <td>The name of the tool.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">created</td>
+        <td>string</td>
+        <td>Timestamp of when the tool was created.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">definition</td>
+        <td>object</td>
+        <td>The definition of the [Tool](./#tool-definition-schema).</td>
+      </tr>
+    </table>
+  </TabItem>
+</Tabs>
+
+
+## Get Tool
+<Badge text="GET /tools/{tool_id}" variant="note" size="large" />
+
+Retrieves details of a specific tool. API key scoped.
+
+<Tabs>
+  <TabItem label="Parameters">
+    <table class="w-full">
+      <tr class="w-full">
+        <th class="w-1/12"></th>
+        <th class="w-11/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">tool_id<br /><span class="text-sm text-red-500">required</span></td>
+        <td>Unique identifier of the tool to retrieve.</td>
+      </tr>
+    </table>
+  </TabItem>
+  <TabItem label="Request Body">
+    <table class="w-full">
+      <tr class="w-full">
+        <th class="w-1/12"></th>
+        <th class="w-1/12"></th>
+        <th class="w-10/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">None</td>
+        <td></td>
+        <td></td>
+      </tr>
+    </table>
+  </TabItem>
+  <TabItem label="Response">
+    <table>
+      <tr>
+        <th class="w-1/12"></th>
+        <th class="w-1/12"></th>
+        <th class="w-10/12"></th>
+      </tr>
+      <tr>
+        <td class="font-mono">toolId</td>
+        <td>string</td>
+        <td>Unique identifier for the tool.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">name</td>
+        <td>string</td>
+        <td>The name of the tool.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">created</td>
+        <td>string</td>
+        <td>Timestamp of when the tool was created.</td>
+      </tr>
+      <tr>
+        <td class="font-mono">definition</td>
+        <td>object</td>
+        <td>A [Tool](./#tool-definition-schema) defintion object for the tool.</td>
+      </tr>
+    </table>
+  </TabItem>
+</Tabs>
+
+## Update Tool
+<Badge text="PUT /tools/{tool_id}" variant="caution" size="large" />
+
+Updates an existing tool. API key scoped.
```

Comment:
```suggestion
Replaces an existing tool. API key scoped.
```

Up to you. Just want to be clear that there's no way to e.g. add a dynamic parameter other than to use GetTool, modify the response, and send back the full object.

Reply:
can you pls elaborate?

Reply:
Even if you only want to change one field, you have to send every field in your request.

Reply:
thx. spelling this out.